cmd/influx/cli/cli.go

```diff
@@ -182,7 +182,13 @@ func (c *CommandLine) Run() error {
 	for {
 		select {
 		case <-c.osSignals:
-			close(c.Quit)
+			// Attempt to gracefully exit
```

Comment:
@jsternberg I'm not sure that's technically the case. If two signals are received in quick succession the scheduler could decide to chose the `case <-c.osSignals` case twice in succession. 

I don't think there is anything that guarantees a closed-channel-receive will always be chosen if there are competing cases available to select over is there? I could be wrong though.